tests/test_workflows/test_validate.py

```diff
@@ -29,32 +29,35 @@ def test_valid_workflow_yaml():
 
 
 @pytest.mark.parametrize(
-    "replace_to_break",
+    "replacee, replacer, error_message_contains",
     [
-        # "version" is required
-        ('version: "2.0"', ""),
```

Comment:
Rewriting this test so that the comments that explain the offense are now formally part of the test (by checking the error message in the `pytest.raises` call).